x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -46,6 +46,15 @@ interface GetAlertParams {
   index?: string;
 }
 
+export const validFeatureIds = ['apm', 'observability', 'siem'] as const;
```

Comment:
How do we distinguish between `observability` and `apm`, etc. Is `apm` a subset of `observability`?

Reply:
Data grid should know where it is.. but yeah I think that's what observability is going for with their "top alerts" rule which looked to be querying `.alerts`..

https://github.com/elastic/kibana/blob/5d95e2e0cd59438e76e7a585eed7ac382f78b057/x-pack/plugins/observability/server/lib/rules/get_top_alerts.ts#L27